Phase1/test/FullTextSearchTest.java

```diff
@@ -0,0 +1,49 @@
+package test;
+
+import src.*;
+import java.util.*;
+import static org.junit.Assert.*;
+import org.junit.Test;
+import org.junit.Before;
+
+public class FullTextSearchTest {
+    private InvertedIndex invertedIndex;
```

Comment:
like previous comment sounds that you should mock these object too.( because you used methods from this objects)